### DISCUSSION TITLE
Correct way to glob .env files?

### DISCUSSION THREAD

**Discussion by bryansoftdev** [vercel/turborepo#7795] [Help] [answered]
### Summary

I just saw the "dotEnv" field does not support globs. How come? 😮

Is this the right way of caching .env files through the use of a glob?

```
"build": {
  "inputs": [$TURBO_DEFAULT$, ".env*"],
},
```
    
Thanks for the help.

**Comment by anthonyshew** [comment, marked as answer]
As a part of the update to 2.0, we've dropped `dotEnv`. Codemod is here: https://turbo.build/repo/docs/reference/turbo-codemod#migrate-dot-env
